src/bin/lpython.cpp

```diff
@@ -138,18 +140,19 @@ int emit_asr(const std::string &infile,
     bool with_intrinsic_modules, CompilerOptions &compiler_options)
 {
     Allocator al(4*1024);
+    LFortran::diag::Diagnostics diagnostics;
     LFortran::Result<LFortran::LPython::AST::ast_t*> r1 = parse_python_file(
-        al, runtime_library_dir, infile);
+        al, runtime_library_dir, infile, diagnostics, false);
     if (!r1.ok) {
         return 1;
     }
     LFortran::LPython::AST::ast_t* ast = r1.result;
 
     LFortran::LocationManager lm;
     lm.in_filename = infile;
-    std::string input = read_file(infile);
+    std::string input = LFortran::read_file(infile);
     lm.init_simple(input);
-    LFortran::diag::Diagnostics diagnostics;
+    diagnostics.diagnostics.clear();
```

Comment:
Do we want to clear it? I think we want to accumulate all messages, such as warnings from the parser, wouldn't we?

Reply:
I left this in, as currently we print the messages from the parser, so that we clear it, so that we don't print it again.